provider/rest/rest.go

```diff
@@ -0,0 +1,65 @@
+package rest
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/containous/mux"
+	"github.com/containous/traefik/log"
+	"github.com/containous/traefik/safe"
+	"github.com/containous/traefik/types"
+	"github.com/unrolled/render"
+)
+
+// Provider is a provider.Provider implementation that provides the UI
+type Provider struct {
```

Comment:
We should change the comment because the provider does not provides `UI` yet, isn't it?